### PR TITLE
Add Restify middleware

### PR DESCRIPTION
**Pull request by Vratislav**
This PR adds Restify middleware to the project.

I have tested it with the example that is provided as a part of this PR.

### CODE REVIEW

**Comment by Vratislav** [comment]
Hello, can I somehow help to get this merged? :)

File: packages/graphql-playground-middleware-restify/src/index.ts [hunk 1]
```diff
@@ -0,0 +1,34 @@
+import {
+  renderPlaygroundPage,
+} from 'graphql-playground-middleware';
```
Comment (timsuchanek) [inline packages/graphql-playground-middleware-restify/src/index.ts]: Please use `renderPlaygroundPage` of the `graphql-playground-html` package

File: packages/graphql-playground-middleware-restify/src/index.ts [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+  req: Request,
+  res: Response,
+  next: () => void,
+) => void
+
+export type Register = (
+  options: MiddlewareOptions,
+) => RestifyPlaygroundMiddleware
+
+const restify: Register = function voyagerRestify(options: MiddlewareOptions) {
```
Comment (timsuchanek) [inline packages/graphql-playground-middleware-restify/src/index.ts]: This is not the GraphQL Voyager ;)

File: packages/graphql-playground-middleware-restify/package.json [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+    "restify": "^6.3.4"
+  },
+  "devDependencies": {
+    "@types/node": "^8.0.47",
+    "@types/restify": "^5.0.6",
+    "typescript": "^2.6.1"
+  },
+  "dependencies": {
+    "graphql-playground-html": "^1.3.6",
+    "graphql-playground-middleware": "^1.2.1-beta.6"
```
Comment (timsuchanek) [inline packages/graphql-playground-middleware-restify/package.json]: please remove the `graphql-playground-middleware` dependency and update `graphql-playground-html` to the latest version

File: packages/graphql-playground-middleware-restify/package.json [hunk 4]
```diff
@@ -0,0 +1,38 @@
+{
+  "name": "graphql-playground-middleware-restify",
+  "version": "1.3.6",
+  "homepage":
+    "https://github.com/graphcool/graphql-playground/tree/master/packages/graphql-playground-middleware-restify",
+  "description":
+    "GraphQL IDE for better development workflows (GraphQL Subscriptions, interactive docs & collaboration).",
+  "contributors": [
+    "Tim Suchanek <tim@graph.cool>",
+    "Johannes Schickling <johannes@graph.cool>",
```
Comment (timsuchanek) [inline packages/graphql-playground-middleware-restify/package.json]: Please add your name!

**Comment by timsuchanek** [comment]
Closing this due to inactivity.

**Pull request closed without merging**